views/projects/showproject.jade

```diff
@@ -144,13 +160,20 @@ div.neumeSection(style = "padding-top : 50px")
                         |  CSV 
                         span.tooltiptext To download the project as a .csv file, click here!
     if (user != null)
-      form(action='/fork',method='post',enctype='application/x-www-form-urlencoded')
-              input(type='text',value='#{project.id}', name='IdOfProject' hidden)
-              input(type='text',value='#{project.name}', name='projectName' hidden)
-              button(type = 'submit' class = 'button button2CSV' class = 'tooltip' style = 'z-index: 10001; margin-top: 20px')
-                i.fa.fa-code-fork
-                |  Fork
-                span.tooltiptext To save the project as your own, click here!
+      .modal(id = "saveCopyModal")
+       .modal-content
+         span.close x
+         form(action='/makeCopy',method='post',enctype='application/x-www-form-urlencoded')
+          h5(style = "font: 18px Avenir, sans-serif; text-align: left;") Enter the name you would like to save the copy under
```

Comment:
I think this heading could be confusing to some people. "Enter a new name for the copy"?